src/common/utility/al-cabinet.ts

```diff
@@ -41,11 +42,12 @@ export class AlCabinet
      *  Instantiates a persistent information cache (uses localStorage), deserializing data from the provided name if it exists.
      *
      *  @param rawName - The name of the data cluster.
+     *  @param syncLatency An optional delay interval between when a change is made and when it is flushed to local storage.
      *
      *  @returns A cabinet instance that can be used to interrogate/update the data.
      */
 
-    public static persistent( rawName:string ):AlCabinet {
+    public static persistent( rawName:string, syncLatency:number = 0 ):AlCabinet {
```

Comment:
I don't see this change is used -- I can see the utility of the mechanism under some circumstances, but what is it for in this context?